x-pack/plugins/observability_solution/observability_onboarding/public/application/quickstart_flows/firehose/create_stack_in_aws_console.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiText, EuiButton, EuiSpacer } from '@elastic/eui';
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import {
+  FIREHOSE_CLOUDFORMATION_STACK_NAME,
+  FIREHOSE_LOGS_STREAM_NAME,
+  FIREHOSE_METRICS_STREAM_NAME,
+} from '../../../../common/aws_firehose';
+import { buildCreateStackAWSConsoleURL } from './utils';
+
+interface Props {
+  encodedApiKey: string;
+  elasticsearchUrl: string;
+  templateUrl: string;
+  isPrimaryAction: boolean;
+}
+
+export function CreateStackInAWSConsole({
+  encodedApiKey,
+  elasticsearchUrl,
+  templateUrl,
+  isPrimaryAction,
+}: Props) {
+  const awsConsoleURL = buildCreateStackAWSConsoleURL({
+    templateUrl,
+    stackName: FIREHOSE_CLOUDFORMATION_STACK_NAME,
+    logsStreamName: FIREHOSE_LOGS_STREAM_NAME,
+    metricsStreamName: FIREHOSE_METRICS_STREAM_NAME,
+    elasticsearchUrl,
+    encodedApiKey,
+  });
+
+  return (
+    <>
+      <EuiText>
+        <p>
+          <FormattedMessage
+            id="xpack.observability_onboarding.firehosePanel.createFirehoseStreamInAWSConsoleDescription"
+            defaultMessage="Review all the pre-filled parameters and submit the form to create a CloudFormation stack from our template. The stack will include a Firehose delivery stream, backup S3 bucket, CloudWatch subscription filter, metrics stream, and necessary IAM roles. Keep this page open, and return once you've submitted the form."
```

Comment:
I find this copy fairly confusing. The user is still on the onboarding page but the copy assumes they have already followed the link and are looking at the AWS console. 

Similar to how the CLI flow instructions first ask the user to "Run the command below..." we should probably start with the first action here as well which is to follow the link, e.g. "Follow the link below to create an Amazon Data Firehose Stream to ingest logs and metrics into Elastic."

Reply:
Very good point, thank you! I'll work with Christian on the copy.